pephubclient/helpers.py

```diff
@@ -84,3 +88,18 @@ def call_client_func(func: Callable[..., Any], **kwargs) -> Any:
         MessageHandler.print_warning(f"PEP already exists. {err}")
     except OSError as err:
         MessageHandler.print_error(f"{err}")
+
+
+def is_registry_path(input_string: str) -> bool:
```

Comment:
Something very similar exists in Looper. Should this, therefore, now be placed in ubiquerg along with `parse_registry_path`?

Reply:
No, this should be part of pephubclient, and looper should now use this function